docs/docker.md

```diff
@@ -0,0 +1,32 @@
+
+## Prerequisites
+1. Docker installed  [(Docker for Mac, Docker for windows, linux, etc)](https://www.docker.com/get-docker)
+
+## Steps to run the git repo unittests from docker
+
+1.  Build the Dockerfile. `` docker build -t moviepy -f Dockerfile . ``
+2.  Get a bash prompt in the moviepy container 
+
+```
+cd tests
+docker run -it -v `pwd`:/tests moviepy bash
+```
+
+3. run the tests.. `` python test_issues.py ``
+
+## Running your own moviepy script from docker
+
+Change directory to where your script is located
+
+If already running, you can connect to the running container.
+
+```
+docker exec -it moviepy python myscript.py
+```
+
+If the container isn't running already
+
+```
+docker run -it moviepy bash
+python myscript.py
+```
```

Comment:
You could add one more condition

This will mount the current directory into /code, and then run the myscript.py file inside of /code. Since this a volume mount, you can edit the code outside of the container, and run inside of the container.
```
docker run -it -v `pwd`:/code moviepy python myscript.py
```